File: NetBeans IDE/Evaluacion 2/PRACTICAS 1-10/EVA2_PRACTICA_9/Python/main.py
print("Ax^2 + Bx + C = 0")
print("A = ", end='', flush=True)
iA = int(input())
print("B = ", end='', flush=True)
iB = int(input())
print("C = ", end='', flush=True)
iC = int(input())
print(iA, end='', flush=True)
print("x^2 + ", end='', flush=True)
print(iB, end='', flush=True)
print("x + ", end='', flush=True)
print(iC, end='', flush=True)
print(" = 0")
dRaiz = iB ** 2 - 4 * iA * iC
dDisc = sqrt(dRaiz)
dX1 = (-iB + dDisc) / (2 * iA)
dX2 = (-iB - dDisc) / (2 * iA)
if dRaiz == 0:
    print("Es un Trinomio Cuadrado Perfecto")
    print("Sus raices son: ", end='', flush=True)
    print("x1 = ", end='', flush=True)
    print(dX1, end='', flush=True)
    print(", x2 = ", end='', flush=True)
    print(dX2)
else:
    if dRaiz > 0:
        print("Es otro tipo de Trinomio")
    else:
        print("No se puede resolver")

File: NetBeans IDE/Evaluacion 2/PRACTICAS 1-10/EVA2_PRACTICA_9/C++/main.cpp
#include <iostream>
#include <sstream>
#include <string>
#include <cstdlib>
#include <cmath>

using namespace std;

// Headers
string toString (double);
int toInt (string);
double toDouble (string);

int main() {
    int iA, iB, iC;
    
    cout << "Ax^2 + Bx + C = 0" << endl;
    cout << "A = ";
    cin >> iA;
    cout << "B = ";
    cin >> iB;
    cout << "C = ";
    cin >> iC;
    cout << iA;
    cout << "x^2 + ";
    cout << iB;
    cout << "x + ";
    cout << iC;
    cout << " = 0" << endl;
    double dRaiz, dDisc, dX1, dX2;
    
    dRaiz = pow(iB, 2) - 4 * iA * iC;
    dDisc = sqrt(dRaiz);
    dX1 = (-iB + dDisc) / (2 * iA);
    dX2 = (-iB - dDisc) / (2 * iA);
    if (dRaiz == 0) {
        cout << "Es un Trinomio Cuadrado Perfecto" << endl;
        cout << "Sus raices son: ";
        cout << "x1 = ";
        cout << dX1;
        cout << ", x2 = ";
        cout << dX2 << endl;
    } else {
        if (dRaiz > 0) {
            cout << "Es otro tipo de Trinomio" << endl;
        } else {
            cout << "No se puede resolver" << endl;
        }
    }
    return 0;
}

// The following implements type conversion functions.
string toString (double value) { //int also
    stringstream temp;
    temp << value;
    return temp.str();
}

int toInt (string text) {
    return atoi(text.c_str());
}

double toDouble (string text) {
    return atof(text.c_str());
}
